Sprint-3/3-mandatory-practice/implement/repeat.js

```diff
@@ -1,5 +1,22 @@
-function repeat() {
-    return "hellohellohello";
+function repeat(word, times) {
+
+    if (times == 0) {
+        return "";
+    }
+
+    if (times == 1) {
+    return word;
+    }
```

Comment:
Why do you need to handle the 0 and 1 cases specially? What would break if you removed these special cases?